website/source/guides/packer-on-cicd/uploading-images-to-artifact.html.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: guides
+sidebar_current: guides-packer-on-cicd-upload-image-to-artifact-store
+page_title: Uploading Images to Artifact Stores
+description: |-
+  ...
+---
+
+# Uploading Images to Artifact Stores
+
+Once the image is generated it will be used by other parts of your operations workflow. For example, it is common to build VirtualBoxes with Packer to be used as base boxes in Vagrant.
+
+On the agent machine install the [AWS Command Line Tool](https://aws.amazon.com/cli/). Since this is a one-time operation, this can be incorporated into the initial provisioning step when installing other dependencies.
+
+```shell
+pip install awscli
```

Comment:
worth calling out the requirements from http://docs.aws.amazon.com/cli/latest/userguide/installing.html ?

```
Requirements

Python 2 version 2.6.5+ or Python 3 version 3.3+
Windows, Linux, macOS, or Unix
```